assets/src/edit-story/app/media/utils/getResourceFromLocalFile.js

```diff
@@ -120,21 +129,48 @@ const getVideoResource = async (file) => {
   });
 };
 
+const createPlaceholderResource = (properties) => {
+  return createLocalResource({ ...properties, isPlaceholder: true });
+};
```

Comment:
What the point of having this as another function here?

Reply:
This is kept in the same style as `createLocalResource` which is also a separate dedicated reason.

Describes quite explicitly what it does, which is what I like about it.